packages/specification/schemas/components/MedicationRequest/CancelRequestMedicationRequest.yaml

```diff
@@ -1,6 +1,7 @@
 type: object
 required:
   - statusReason
+  - extension
```

Comment:
The extension field (line 34-38) currently contains only DMResponsiblePractitioner items, but example primary-care/acute/nominated-pharmacy/medical-prescriber/cancellations/1-Process-Request-Cancel-200_OK.json has an extension containing a Extension-DM-PrescriptionType, but no Extension-DM-ResponsiblePractitioner.

It might be worth double checking what that field can contain before we mark it as mandatory

Reply:
https://simplifier.net/packages/uk.nhsdigital.r4/2.4.5/files/566259/~details Looking down at responsiblePractitioner (eps-5) it looks like you might be right as it states to only supply this when the requester can't be used. So i'll revert this one too.